packages/discord.js/tsconfig.json

```diff
@@ -3,8 +3,12 @@
   "compilerOptions": {
     // Output Formatting
     "pretty": false,
+    "lib": ["ESNext"],
 
     // Completeness
-    "skipDefaultLibCheck": true
-  }
+    "skipDefaultLibCheck": true,
+    "allowJs": true,
+    "outDir": "./dist"
+  },
+  "include": ["src/**/*", "typings/**/*"]
```

Comment:
This will compile to `dist/src`. So why not move types inside src and auto gen types?

Reply:
If we autogen type files we'll end up with definitions that are worse than our manual definitions, since they're pulled from js.